CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is mostly based on [Keep a Changelog](https://keepachangelog.com/en/1
 
 ## [18.x.x]
 ### Changed
-- Change shortcut descriptions. (#1669)
```

Comment:
Wait, why is this removed?

Reply:
I guess because I thought about merging it after the release and rebasing this anyway